src/containers/course-section/CourseSectionContainer.tsx

```diff
@@ -379,49 +346,19 @@ const CourseSectionContainer: FC<SectionProps> = ({
     </MenuItem>
   ))
 
+  const onDeleteSection = () => {
+    setSectionsItems(sections.filter((item) => item.id !== sectionData.id))
```

Comment:
I suggest using separate constant for `sections.filter((item) => item.id !== sectionData.id)` for better readability

Reply:
It's quite a huge file so ,in my opinion, it's better to stay like this